index.js

```diff
@@ -225,6 +229,41 @@ async function main (argv) {
   }
   process.stdout.write('OK')
 
+  //
+  //  Initialize tsconfig.json when react_ts 
+  //
+  if (templateName === 'react_ts') {
+    try {
+      process.stdout.write('\nCreating tsconfig....')
+      await exec(
+        'npx tsc --init --declaration --allowJs --emitDeclarationOnly --jsx react-jsx --lib "dom","dom.iterable","esnext" --outDir dist'
+      )
+    } catch (err) {
+      process.stderr.write(
+        `\nFailed to create tsconfig: ${err.stack ?? err.message}\n`
+      )
+      process.exit(1)
+    }
+
+    process.stdout.write('OK')
+
+    try {
+      process.stdout.write('\nSetting up TS configuration...')
+      await fs.writeFile(
+        'globals.d.ts',
+        "declare module 'socket:os'; \ndeclare module 'socket:test'; \ndeclare module 'socket:console'; \ndeclare module 'socket:process';"
+      )
+    } catch (err) {
+      process.stderr.write(
+        `\nFailed to create global.d.ts: ${
+          err.stack ?? err.message
+        }. Please do it manually :)\n`
```

Comment:
It might be better to direct the user to report an issue.

Reply:
I propose something like that :
```
process.stderr.write(
        `\nFailed to create global.d.ts: ${
          err.stack ?? err.message
        }. Please report this issue here: https://github.com/socketsupply/create-socket-app/issues \n`
      )
```

Reply:
This looks great, aside from the leading `\n` :)